ui-frontend/packages/catalog-ui-search/package.json

```diff
@@ -101,7 +101,7 @@
     "font-awesome": "4.7",
     "geo-convex-hull": "1.2.0",
     "geolib": "^3.2.1",
-    "geospatialdraw": "0.4.5",
+    "geospatialdraw": "file://opt/geospatialdraw/geospatialdraw-0.4.6.tgz",
```

Comment:
Reminder that this change should not be merged in.

Reply:
Will update to 0.4.6 once geospatial draw is published